src/core/net/ip6_address.hpp

```diff
@@ -269,6 +272,24 @@ class Address : public otIp6Address
      */
     bool IsIidReserved(void) const;
 
+    /**
+     * This method gets the IPv6 address locator.
+     *
+     * @returns RLOC16 or ALOC16.
+     *
+     */
+    inline uint16_t GetLocator(void) const { return HostSwap16(mFields.m16[7]); }
```

Comment:
```suggestion
    uint16_t GetLocator(void) const { return HostSwap16(mFields.m16[7]); }
```
No need to explicitly declare as inline.